src/styles/shared/reset.scss

```diff
@@ -201,14 +201,14 @@ html {
 }
 body {
   color: hsla(0, 0%, 0%, 0.8);
-  font-family: "Helvetica Neue", serif;
+  font-family: 'Roboto', 'Helvetica Neue', serif;
```

Comment:
This will overwrite the font-family on every page - not what we want. Please add this style rule inside `.layout.privacy-tos-page`

Reply:
Totally agreed. Changed as instructed.